addons/l10n_it_edi/tests/test_edi_export.py

```diff
@@ -495,3 +495,33 @@ def test_export_foreign_currency(self):
         })
         invoice.action_post()
         self._assert_export_invoice(invoice, 'export_foreign_currency_global_discount.xml')
+
+    def test_export_invoice_with_two_downpayments(self):
```

Comment:
I think we need some `@freeze_time` or any way to fix the date in time.

Reply:
The payment date is checked against... today.
![immagine](https://github.com/user-attachments/assets/12ea3657-8e4c-4d34-a29b-ba8a97167167)

Reply:
sorry didn't notice

good to catch before merging 🙏  fortunately it was merged a different day